submissions/OlegPopovych/html_css_popup/index.html

```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link href="https://fonts.googleapis.com/css?family=Lato:100,300,400,700,900" rel="stylesheet">
+	<link rel="stylesheet" href="css/main.css">
+	<link rel="shortcut icon" type="image/png" href="img/favicon.png">
+	<title>Popup</title>
+</head>
+
+<body>
+	<nav class="nav">
+		<ul class="nav__list">
+			<li class=" nav__list--item">
+				<h1 class="nav__title">Hello Kottans!</h1>
+			</li>
+			<li class="nav__list--item">
+				<a class="nav__link" href="#">
+					<span>Mail</span>
+				</a>
+			</li>
+			<li class="nav__list--item">
+				<a class="nav__link" href="#">
+					<span>Photos</span>
+				</a>
+			</li>
+			<li class="nav__list--item relative">
+				<input class="popup__check nav__label nav__input" type="checkbox" id="check-1" />
+				<label class="nav__label " for="check-1">
+					<a class="icon" alt="Menu opener">
+						<svg fill="none" height="24" viewBox="0 0 24 24" width="24" class="icon icon-rotate"
+							xmlns="http://www.w3.org/2000/svg">
+							<path fill="#CBD5E4" d="M4 6H20M4 12H20M4 18H20" stroke="#CBD5E4" stroke-linecap="round"
+								stroke-linejoin="round" stroke-width="2" />
+						</svg>
+					</a>
+				</label>
+				<div class="popup">
+					<ul class="popup__menu">
+						<li class="popup__menu--item">
+							<a class="popup__link" href="#">
+								<img class="icon" src="img/blogs.png" alt="Blogs" />
```

Comment:
For icons it's better set an empty alt attribute, read more [here](https://accessibility.psu.edu/images/imageshtml/#empty)

Reply:
I'm set empty attribute for icons